frontend/ios/DroidKaigi 2019/Scenes/FloorMap/FloorMapContentViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  FloorMapContentViewController.swift
+//  DroidKaigi 2019
+//
+//  Created by 417.72KI on 2019/01/31.
+//
+
+import UIKit
+import XLPagerTabStrip
+
+class FloorMapContentViewController: UIViewController {
+
+    var floor: Floor?
```

Comment:
I'm not good at Swift. But I think If always floor injected by the system. And field don't become null. When Kotlin code, We uses `Floor!` type (Kotlin `lateinit` property)

Reply:
I'm afraid of crash by forgetting to set floor in caller view.
[FYI] Some projects set lint to give warning with `Hoge!` type (In swift, we call them **IUO**).

But in this case, you're right because it is *Logic Error* and app should crash.
So I will fix it.